orchestrator/api/community_skills.py

```
@@ -0,0 +1,231 @@
+    skill_name = payload.get("skill_name", "").strip()
+    force = payload.get("force", False)
+    if not skill_name:
+        raise HTTPException(400, "skill_name is required")
+
+    import requests
+
+    # 1. Fetch from skills.sh
+    try:
+        resp = requests.get(f"{SKILLS_SH_API}/skills/{skill_name}", timeout=15)
+        if resp.status_code != 200:
+            raise HTTPException(404, f"Skill '{skill_name}' not found on skills.sh")
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)